src/main/kotlin/app/revanced/patches/youtube/player/overlaybuttons/OverlayButtonsPatch.kt

```diff
@@ -74,152 +102,132 @@ object OverlayButtonsPatch : BaseResourcePatch(
-                        attribute.textContent = "@+id/speed_dialog_button"
+                node.getAttributeNode("yt:layout_constraintRight_toLeftOf")
+                    ?.let { attribute ->
+                        if (attribute.textContent == "@id/fullscreen_button") {
+                            attribute.textContent = "@+id/speed_dialog_button"
+                        }
                     }
-                }
 
-                it.getAttributeNode("android:id")?.let { attribute ->
-                    // Adjust Fullscreen Button size and padding
-                    arrayOf(
-                        "speed_dialog_button",
-                        "copy_video_url_button",
-                        "copy_video_url_timestamp_button",
-                        "always_repeat_button",
-                        "external_download_button",
-                        fullscreenButtonId
-                    ).forEach { targetId ->
-                        if (attribute.textContent.endsWith(targetId)) {
-                            arrayOf(
-                                "0.0dip" to arrayOf("paddingLeft", "paddingRight"),
-                                bottomPadding to arrayOf("paddingBottom"),
-                                "48.0dip" to arrayOf("layout_height", "layout_width")
-                            ).forEach { (replace, attributes) ->
-                                attributes.forEach { name ->
-                                    it.getAttributeNode("android:$name")?.textContent = replace
-                                }
-                            }
+                // Adjust TimeBar and Chapter bottom padding
+                arrayOf(
+                    "@id/time_bar_chapter_title" to "16.0dip",
+                    "@id/timestamps_container" to "14.0dip"
+                ).forEach { (id, replace) ->
+                    node.getAttributeNode("android:id")?.let { attribute ->
+                        if (attribute.textContent == id) {
+                            node.getAttributeNode("android:paddingBottom").textContent =
+                                replace
                         }
                     }
                 }
 
-                if (WiderBottomPadding == false) {
-                    // Adjust TimeBar and Chapter bottom padding
-                    arrayOf(
-                        "@id/time_bar_chapter_title" to "14.0dip",
-                        "@id/timestamps_container" to "12.0dip"
-                    ).forEach { (id, replace) ->
-                        it.getAttributeNode("android:id")?.let { attribute ->
-                            if (attribute.textContent == id) {
-                                it.getAttributeNode("android:paddingBottom").textContent = replace
-                            }
-                        }
+                // Adjust layout for fullscreen button stub
+                if (node.getAttribute("android:id") == "@id/youtube_controls_fullscreen_button_stub") {
```

Comment:
Yes, you can also test with my patches before merging, maybe you'll see something I couldn't during testing. But this change is old and no reports for any issues from other people.

Reply:
It worked because it has try - catch for old fullscreen, so it adjusts the old fullscreen anyway but only if it exists because of try - catch and then it just uses if statements to check if id present for new fullscreen stub and if not then it does nothing. But maybe I'm missing something. This code could be refactored and made more readable but not sure how to make it better. If you want, you can merge it and then edit it how you think would be better.

Reply:
Or maybe I misundertood you, there next if statement that adjusts old fullscreen, as all of them were ending with "_button"
```kotlin
if (node.getAttribute("android:id").endsWith("_button")) {
```